File: C/URI 1060.c
#include <stdio.h>
int main(){
  int i, contador = 0;
  float valor;
  for(i = 1; i <= 6; i++){
      scanf("%f",&valor);
      if(valor >= 0){
          contador = contador + 1;
      }
  }
  printf("%d valores positivos\n",contador);
  return 0;
}

File: Java/URI 1060.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
  public static void main(String[] args) throws IOException {
    Scanner sc = new Scanner(System.in);
    int contador = 0;
    for(int i = 0; i < 6; i++){
        double valor = sc.nextDouble();
        if(valor > 0){
            contador = contador + 1;
        }
    }
    System.out.println(contador + " valores positivos");
  }
}